transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -304,14 +304,15 @@ private void processReady(EpollEventArray events, int ready) {
                 final long ev = events.events(i);
 
                 AbstractEpollChannel ch = channels.get(fd);
-                if (ch != null && ch.isOpen()) {
-
+                if (ch != null) {
                     AbstractEpollUnsafe unsafe = (AbstractEpollUnsafe) ch.unsafe();
 
-                    // We need to check if the channel is still open before try to trigger the
-                    // callbacks.
-                    // See https://github.com/netty/netty/issues/3443
-                    if ((ev & Native.EPOLLRDHUP) != 0 && ch.isOpen()) {
+                    // First check if EPOLLIN was set, in this case we not need to check for
+                    // EPOLLRDHUP as with EPOLLIN already handled the connection-reset case.
```

Comment:
we not need -> we do not need

I don't understand the 'as with EPOLLIN...' clause.  Could you rephrase or fix it?